components/header.ftd

```diff
@@ -8,6 +8,7 @@ module site-data: site-data
 module user-data: user-data
 optional string product:
 ds.gap margin: $ds.spaces.vertical-gap.zero
+optional boolean $sidebar: false
```

Comment:
Are you using this anywhere? @ishaan701

Reply:
Its usage is present in other files

Reply:
Is it used in `header` component itself? @ishaan701

Reply:
yes